charts/aws-ebs-csi-driver/templates/node.yaml

```diff
@@ -65,6 +65,10 @@ spec:
           env:
             - name: CSI_ENDPOINT
               value: unix:/csi/csi.sock
+            - name: CSI_NODE_NAME
+              valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
```

Comment:
also run `make generate-kustomize` (definitely we need a CI job for this, same as how we have make verify for keeping vendor dir up to date)

Reply:
done in new PR #855